src/sqs/sqs.ts

```diff
@@ -306,10 +306,10 @@ export class SqsQueue<MessagePayload extends BaseMessagePayload> {
 				messageId: message.MessageId,
 				executionId: uuidv4(),
 				queue: this.queueName,
-				jiraHost: payload?.jiraHost,
-				installationId: payload?.installationId,
-				gitHubAppId: payload?.gitHubAppConfig?.gitHubAppId,
-				webhookId: payload?.webhookId,
+				jiraHost: payload.jiraHost,
+				installationId: payload.installationId,
+				gitHubAppId: payload.gitHubAppConfig?.gitHubAppId,
+				webhookId: payload.webhookId,
```

Comment:
Are we sure these are always there? :susp: Backfilling, for instance, doesn't have webhookId, and this file looks like some common one...

Reply:
Ah wait, probably that would be the same outcome anyway, which is "undefined"

Reply:
That is right these properties could be undefined but payload will never be undefined here